docs/source/simulated_populations/index.rst

```diff
@@ -65,6 +65,12 @@ The following table provides the SHA-256 checksum for the larger-scale simulated
   * - Location
     - File
     - SHA-256 checksum
+  * - US
+    - pseudopeople_simulated_population_usa_2_0_0.zip
```

Comment:
Did we decide that 2.0.0 is actually the version number we want to use for the data?

Reply:
for better or worse, we did